lib/takeaway.rb

```diff
@@ -0,0 +1,58 @@
+require 'twilio-ruby'
+
+class Takeaway
+
+  attr_reader :items, :shoppingcart
+
+  def initialize
+    @items = 
+    { "Margarita" => 8.99, "Vegetarian" => 9.99, "Pepperoni" => 10.99, 
+    "Cheesecake" => 3.99, "Water" => 0.99 }
+
+    @shoppingcart = []
+  end
+
+  def read_menu
+    p "What would you like to order? We currently have...."
+    p @items
+  end
+
+  def add(item)
+    @shoppingcart << @items[item.capitalize] 
+    # blocker: to convert it over to being a symbol, 
+    # this wasn't needed actually. hash was the wrong values.
+    "#{item.capitalize} added to your basket. The total currently is £#{total}"
+  end
+
+  def total
+    @shoppingcart.reduce(:+).round(2)
+  end
+
+  def time
+    time = Time.now + 3600 # seconds in an hour
+    time.strftime("%k:%M")
+  end
+
+  def sendtext
+   
+    # removed my sid/auth/mobile numbers for upload
+    account_sid = 'ACf213fd0e0968cad4cf1d53ef5e2fe301'
+    auth_token = 'f3a00b928bb594b9847b5253a4d5092c'
+    client = Twilio::REST::Client.new(account_sid, auth_token)
+
+    from = '+447723565836' # Your Twilio number
+    to = '+447709018333' # Your mobile phone number
+
```

Comment:
This is good but look into how to hide your personal details. I was told this morning that creating an ENV file on your local computer is how you can add your personal details without having to post them on GitHub.